src/store/reducers/transactions.js

```diff
@@ -60,57 +57,52 @@ const transactions = (state = initialState, action) => { // eslint-disable-line
         pending: addNewTransactions(state.pending, action.data.confirmed),
       };
     }
-    // TODO can be remove after move send (create) tx to utils file
-    // istanbul ignore next
+
+    // Stored the signed transaction to be used for broadcasting or downloading
     case actionTypes.transactionCreatedSuccess:
       return {
         ...state,
-        transactionsCreated: [...state.transactionsCreated, action.data],
+        signedTransaction: action.data,
       };
-    // TODO can be remove after move send (create) tx to utils file
-    // istanbul ignore next
-    case actionTypes.transactionCreatedError: {
-      const { message = 'The transaction failed', name = 'TransactionFailedError' } = action.data;
+
+    // Stores the transaction signature error. This error is thrown at the time of
+    // creating the raw transaction object or signing it using Lisk Element / Bitcoin lib.
+    case actionTypes.transactionCreatedError:
```

Comment:
`transactionSignError` is a better name in my opinion

Reply:
Renamed.